data-structures-linked-lists/remove-tail.js

```diff
@@ -0,0 +1,13 @@
+/* exported removeTail */
+
+function removeTail(list) {
+  if (list.next === null) return;
+  let before = list;
+  let after = list.next;
+  while (after.next != null) {
+    before = after;
+    after = after.next;
+  }
+  before.next = null;
+  return list;
```

Comment:
`removeTail` should not return anything:

---

![image](https://user-images.githubusercontent.com/16143771/156837786-1bc86210-b7f7-4d21-a77f-d7b4b4f6d869.png)